src/Components/Gene/index.tsx

```diff
@@ -6,10 +6,10 @@ import Artists from "./Artists"
 import GeneArtworks from "./GeneArtworks"
 
 interface Filters {
-  for_sale: boolean
-  dimension_range: string
-  price_range: string
-  medium: string
+  for_sale?: boolean
+  dimension_range?: string
+  price_range?: string
+  medium?: string
```

Comment:
This makes these typed as optional to e.g. the `onStateChange`  callback as well, which isn’t really true. I _think_ you really only want these to be optional for the input props, right? If so, you can use do the following:

```ts
interface Props {
  filters?: Partial<Filters>
}
```

Reply:
Oh, nice! Yep, that's right.